packages/truffle-db/test/truffle-projects/solidity-magic-square/README.md

```diff
@@ -0,0 +1,4 @@
+solidity-magic-square
+=====================
```

Comment:
I don't think this should get duplicated here. Can this be a submodule perhaps, linking back to the original repo at github.com/trufflesuite/solidity-magic-square?

Reply:
a plain `git clone` might be fine also

Reply:
Yea, the PR to solidity-magic-square broke it spectacularly. I'm looking at a git-clone option w/ a tag or branch specifically for being a fixture for truffle-db.